docs/questionRuntime/index.md

```diff
@@ -0,0 +1,60 @@
+# The Python Environment for Questions
+
+This page is meant for discussion of details related to the Python environment that questions are run in, along with how to work within this environment for question development. Are you creating questions and finding that PrairieLearn does not have the Python libraries you need? Don't panic! There are a few different options you have for getting the necessary libraries installed that will be discussed below.
+
+## General information
+
+All `server.py` files for questions are executed in a Docker container created from the `prairielearn/prairielearn` image. This image includes the Python version that is bundled with the [latest version of Miniconda](https://docs.conda.io/en/latest/miniconda.html), as well as the packages in [`images/plbase/python-requirements.txt`](https://github.com/PrairieLearn/PrairieLearn/blob/master/images/plbase/python-requirements.txt). The following are basic ways to get additional information about the Python environment.
+
+- To see which packages are installed, run `pip list` in the docker container.
+
+- To run a command line version of this Python environment, you may start it with the following command:
+
+  ```
+  docker run -it --rm prairielearn/prairielearn /bin/bash
+  ```
+
+- To access files on the disk or in `serverFilesCourse` within `server.py`, get the directory from `data["options"]["question_path"]` and similar entries from the `data["options"]` dictionary.
```

Comment:
```suggestion
- To access files on the disk or in `serverFilesCourse` within `server.py`, get the directory from `data["options"]["server_files_course_path"]` and similar entries from the `data["options"]` dictionary.
```

_That said_, I'm not sure if this actually belongs here - if this section is about "general information" about the Python runtime, why are we discussing course-specific files?

Reply:
Removed here https://github.com/PrairieLearn/PrairieLearn/pull/3892/commits/32650f04448a9aa575385dba10889f983dd1f81a